api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -332,4 +354,23 @@ private User initializeUserIfNeeded() {
       return ResponseEntity.status(e.getCode()).build();
     }
   }
+
+  @Override
+  public ResponseEntity<Void> submitIdVerification(IdVerificationRequest request) {
+    Person person = blockscoreService.createPerson(
+      request.getFirstName(), request.getLastName(),
+      blockscoreService.createAddress(
+        request.getStreetLine1(), request.getStreetLine2(),
+        request.getCity(), request.getState(), request.getZip()
+      ),
+      request.getDob(),
+      request.getDocumentType(), request.getDocumentNumber()
+    );
+    User user = userProvider.get();
+    user.setBlockscoreId(person.getId());
```

Comment:
So if this method gets called when there's already a blockscore ID for the user... we overwrite it? (I assume that's OK?)

Reply:
Definitely worth looking into. Added a TODO.